.github/workflows/linux.yml

```diff
@@ -325,10 +325,10 @@ jobs:
         sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/3bf863cc.pub
         sudo add-apt-repository "deb http://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/ /"
         sudo apt-get install -y software-properties-common
-        sudo add-apt-repository ppa:ginggs/deal.ii-9.4.0-backports
+        sudo add-apt-repository ppa:ginggs/deal.ii-9.6.0-backports
         sudo apt-get update
         sudo apt-get install -yq --no-install-recommends \
-            cuda-toolkit-11-8 \
+            cuda-toolkit-12-9 \
```

Comment:
Shouldn't we rather test the minimum we want to support?

Reply:
I think CUDA 12.1 will be enough for `clang`, according to its [prerequisites](https://llvm.org/docs/CompileCudaWithLLVM.html#prerequisites) and the CI warning.
```
clang++-18: warning: CUDA version is newer than the latest supported version 12.3 [-Wunknown-cuda-version]
```

I don't know about any prerequisites for nvcc+gnu.